turbo/execution/eth1/ethereum_execution.go

```diff
@@ -161,6 +161,15 @@ func (e *EthereumExecutionModule) ValidateChain(ctx context.Context, req *execut
 		}, nil
 	}
 	defer e.semaphore.Release(1)
+
+	// Update the last new block seen.
+	// This is used by eth_syncing as an heuristic to determine if the node is syncing or not.
+	if err := e.db.Update(ctx, func(tx kv.RwTx) error {
```

Comment:
it must be in individual rwtx? for speed? 
if yes, yes - then maybe use UpdateNoSync() here.

Reply:
no, it is so that `eth_syncing` can see before the full function returns (there is a bigger rwtx later)